perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -2108,6 +2109,9 @@ sub fetch_all_bodies : Private {
     my ($self, $c ) = @_;
 
     my @bodies = $c->model('DB::Body')->translated->all_sorted;
+    if ( my @cobrand_bodies = $c->cobrand->call_hook('admin_fetch_all_bodies') ) {
```

Comment:
This needs `@bodies` passing to it, looks like, and then the `zurich` `if` below can presumably be removed.